bridges/otellogrus/hook.go

```diff
@@ -183,6 +183,30 @@ func convertFields(fields logrus.Fields) []log.KeyValue {
 	return kvs
 }
 
+func convertSeverity(level logrus.Level) log.Severity {
+	switch level {
+	case logrus.PanicLevel:
+		// PanicLevel is not supported by OpenTelemetry, use Fatal4 as the highest severity.
+		return log.SeverityFatal4
+	case logrus.FatalLevel:
+		return log.SeverityFatal
+	case logrus.ErrorLevel:
+		return log.SeverityError
+	case logrus.WarnLevel:
+		return log.SeverityWarn
+	case logrus.InfoLevel:
+		return log.SeverityInfo
+	case logrus.DebugLevel:
+		return log.SeverityDebug
+	case logrus.TraceLevel:
+		return log.SeverityTrace
+	default:
+		// If the level is not recognized, use Trace as the lowest severity.
+		// we should never reach this point as logrus only uses the above levels.
+		return log.SeverityTrace
```

Comment:
```suggestion
		return log.SeverityUndefined
```

Reply:
fixed. Tbh, I'm not sure if it's a safe fallback but I also hesitated between `log.SeverityUndefined` and `log.SeverityTrace` for this particular case

Reply:
Undefined looks better, as it better indicates we're missing a level.